tests/config/test_material.py

```diff
@@ -0,0 +1,145 @@
+from .common import TestConfig, test_data
+from hexrd.config.material import TTHW_DFLT, DMIN_DFLT
+from hexrd.config.utils import get_exclusion_parameters
+
+
+reference_data = \
+"""
+material:
+  definitions: %(existing_file)s
+#  active: # not set to test error
+---
+material:
+  definitions: %(nonexistent_file)s
+  active: ruby
+  tth_width: 0.2
+---
+material:
+#  definitions: # not set to test inheritance
+  active: CeO2
+use_saved_hkls: true
+---
+# cfg: 3
+material:
+  reset_exclusions: false
+---
+# cfg 4
+material:
+  dmin: 1.0
+  dmax: 3.0
+---
+# cfg 5
+material:
+  tthmin: 1.5
+  tthmax: 5.0
+  min_sfac_ratio: 0.02
+---
+# cfg 6
+material:
+  sfacmin: 0.01
+  sfacmax: 0.99
+---
+# cfg 7
+material:
+  pintmin: 0.05
+  pintmax: 0.95
+"""
+
+
+class TestMaterialConfig(TestConfig):
+
+
+    @classmethod
+    def get_reference_data(cls):
+        return reference_data % test_data
+
+
+    def test_definitions(self):
+        self.assertEqual(
```

Comment:
I like the regular `assert ...` statements we use in pytest. But I guess we need to use these in the unittests? It's okay, we can leave them.

Reply:
I like the simpler assert better also, and for new tests I plan to use the pytest interface.  I'm not sure if you can mix the unittest asserts and regular asserts, so I just kept it consistent in the file.